Frameworks/AddressBook/ABMultiValueInternal.mm

```diff
@@ -159,4 +159,71 @@ - (CFIndex)getIndexForIdentifier:(ABMultiValueIdentifier)identifier {
     return index;
 }
 
+- (_ABMultiValue*)createMutableCopy {
```

Comment:
(and therefore call `mutableCopy` later)

---
In reply to: [75725849](https://github.com/Microsoft/WinObjC/pull/893#discussion_r75725849) [](ancestors = 75725849)